apps/autonolas-registry/common-util/Details/DetailsSubInfo/RewardsSection.tsx

```diff
@@ -19,17 +22,22 @@ const RewardColumn = ({ title, statistic }: { title: string; statistic: string }
   </Col>
 );
 
-export const RewardsSection = memo(function RewardsSection({
+type RewardsSectionProps = { reward: bigint; topUp: bigint };
+export const RewardsSection: FC<RewardsSectionProps> = memo(function RewardsSection({
   reward,
   topUp,
-}: {
-  reward: bigint;
-  topUp: bigint;
 }) {
   return (
-    <Row>
-      <RewardColumn title={'Claimable Reward'} statistic={`${rewardsFormatter(reward, 4)} ETH`} />
-      <RewardColumn title={'Claimable Top-Up'} statistic={`${rewardsFormatter(topUp, 2)} OLAS`} />
-    </Row>
+    <Flex vertical gap={4}>
+      <Row>
+        <RewardColumn title={'Claimable Reward'} statistic={`${rewardsFormatter(reward, 4)} ETH`} />
+        <RewardColumn title={'Claimable Top-Up'} statistic={`${rewardsFormatter(topUp, 2)} OLAS`} />
+      </Row>
+      <Row>
+        <Link href="https://tokenomics.olas.network/donate">
+          Make donation {UNICODE_SYMBOLS.EXTERNAL_LINK}
+        </Link>
```

Comment:
```suggestion
        <a href="https://tokenomics.olas.network/donate">
          Make donation {UNICODE_SYMBOLS.EXTERNAL_LINK}
        </a>
```

Reply:
shouldn't it also have `target="_blank"`? @mohandast52

Reply:
hmm, we open a new tab here when click "guide" https://tokenomics.olas.network/bonding-products

and why to have the `EXTERNAL_LINK` symbol then?